gatsby-browser.js

```diff
@@ -137,6 +137,7 @@ export const onClientEntry = () => {
         addScript({
```

Comment:
Actually we are not using this GTM
Our current GTM container ID, GTM-NF7884S


We can cleanup the old gtm